src/qt/updateassetsdialog.cpp

```diff
@@ -355,35 +355,37 @@ void UpdateAssetsDialog::updateAsset() {
         return;
     }
 
-    CTxDestination ownerAddress = CTxDestination(assetData.ownerAddress);
-    if (!IsValidDestination(ownerAddress)) {
-        QMessageBox msgBox;
-        msgBox.setText("ERROR: Invalid owner address");
-        msgBox.setStandardButtons(QMessageBox::Ok);
-        msgBox.exec();
-        return;
-    }
+    if (!Params().IsRootAssetsActive(::ChainActive().Tip())) {
```

Comment:
this if from line 358 to line 388 is almost identical with assetsdialog.cpp minAsset() line 331 to 361. Can we extract this out to a common function?